src/components/scores/export.cjsx

```diff
@@ -19,7 +19,7 @@ ScoresExport = React.createClass
 
   getInitialState: ->
     downloadUrl: null
-    forceDownloadUrl: null
+    finalDownloadUrl: null
     lastExported: null
     tryToDownload: false
     downloadedSinceLoad: false
```

Comment:
If the values are `falsey`  I don't think you'll need to specify them in `getInitialState`.  For instance if you just return `{}` here and then later call `@state.downloadUrl` it'll be `undefined` which will evaluate to false.  i.e. `if @state. finalDownloadUrl` will not match.